### PR TITLE
CSRF Testing

### PR DESCRIPTION
**Pull request by jerben**

### CODE REVIEW

**Comment by aronerben** [comment]
@jerben Please review!

File: sihl/random/random.ml [hunk 1]
```diff
@@ -1,14 +1,18 @@
 module Sig = Sig
 
 module Service : Sig.SERVICE = struct
-  let rec rand result n =
+  let rec random_char' result n =
```
Comment (jerben) [inline sihl/random/random.ml:4]: @aronerben Why not `chars`?

File: sihl/random/random.ml [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
     if n > 0
-    then rand Base.(result ^ Char.to_string (Random.ascii ())) (n - 1)
+    then random_char' (List.cons (Char.chr (Random.int 255)) result) (n - 1)
     else result
   ;;
 
+  let random_bytes ~bytes = random_char' [] bytes
```
Comment (jerben) [inline sihl/random/random.ml:10]: @aronerben Why not `bytes`?

File: sihl/random/random.ml (hunk 1 above)
Comment (aronerben) [reply sihl/random/random.ml]: @jerben Done

File: sihl/random/random.ml (hunk 2 above)
Comment (aronerben) [reply sihl/random/random.ml]: @Done

**Comment by jerben** [comment]
@aronerben Approved, you can merge :)

**Comment by aronerben** [comment]
@jerben CSRF finally in master! :tada:

**Comment by jerben** [comment]
@aronerben Congrats! :+1:

**Pull request merged**